src/Idris/Syntax.idr

```diff
@@ -269,6 +270,56 @@ mutual
        NFMetavarThreshold : Nat -> Directive
        SearchTimeout : Integer -> Directive
 
+  directiveList : List Directive
+  directiveList =
+      [ (Hide (UN "")), (Logging Nothing), (LazyOn False)
+      , (UnboundImplicits False), (AmbigDepth 0)
+      , (PairNames (UN "") (UN "") (UN "")), (RewriteName (UN "") (UN ""))
+      , (PrimInteger (UN "")), (PrimString (UN "")), (PrimChar (UN ""))
+      , (PrimDouble (UN "")), (CGAction "" ""), (Names (UN "") [])
+      , (StartExpr (PRef EmptyFC (UN ""))), (Overloadable (UN ""))
+      , (Extension ElabReflection), (DefaultTotality PartialOK)
+      , (PrefixRecordProjections True), (AutoImplicitDepth 0)
+      , (NFMetavarThreshold 0), (SearchTimeout 0)
+      ]
+
+  isPragma : Directive -> Bool
+  isPragma (CGAction _ _) = False
+  isPragma _              = True
+
+  export
+  pragmaTopics : String
+  pragmaTopics =
+    show $ vsep $ map (((<++>) "+") . pretty {ann=()} . showDirective) $ filter isPragma directiveList
+    where
+      showDirective : Directive -> String
+      showDirective (Hide _)             = "%hide term"
```

Comment:
This should be name I think.

Reply:
Yep, that also matches what the parser wants it to be. Thanks for spotting it, I'll fix that  : )